dune

```diff
@@ -2,3 +2,5 @@
  (targets graphql_schema.json)
  (mode promote)
  (action (with-stdout-to graphql_schema.json (run %{exe:src/app/graphql_schema_dump/graphql_schema_dump.exe}))))
+
+(data_only_dirs opam_switches)
```

Comment:
I think you want
```
(dirs :standard \ opam_switches)
```
to ensure that dune won't do anything with the directory. You probably don't want it to copy your entire opam switch to `_build` at any point.